src/core/public/utils/index.ts

```diff
@@ -6,6 +6,5 @@
  * Side Public License, v 1.
  */
 
-export { Sha256 } from './crypto';
```

Comment:
I wonder if we should re-export it from `@kbn/crypto-browser` to avoid the bundle bump in the plugins using it?

Either that or we move the library to the ui-shared-deps-src library. What do you think?

Reply:
Since the idea long term is to get rid of `src/core/public/index.ts` altogether, we are going for the "import directly from packages" approach. Then, on a later phase, we can add `@kbn/crypto-browser` (and other packages that are commonly used) to the `ui-shared-deps-src`, so that they are only loaded once.